src/main/kotlin/no/nav/dagpenger/kalkulator/BehovStarter.kt

```diff
@@ -38,7 +38,7 @@ class BehovStarter(
                 "Failed to run behov. Response message ${response.responseMessage}. Error message: ${result.error.message}"
             )
             is Result.Success ->
-                response.headers["Location"].first()
+                response.headers["Location"].first().replaceFirst("/v1/", "/")
```

Comment:
denne er litt blæ, dp-proxy skriver ikke om Location header fra regel-api og original Location header treffer ikke riktig gjennom proxy igjen. Lurer på å lage noe i dp-proxy, men funker for nå.